### PR TITLE
PUBDEV-7826: Release notes for 3.32.0.1 

### PR DESCRIPTION
**Pull request by hannah-tillman**

### CODE REVIEW

File: Changes.md [hunk 1]
```diff
@@ -2,6 +2,94 @@
...
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7760'>PUBDEV-7760</a>] -         H2O no longer uses lazy-loading for sequential zip parse.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7762'>PUBDEV-7762</a>] -         Updated model_type argument names for Rulefit in R.
+</li>
+</ul>
+    
+#### New Feature
+<ul>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7241'>PUBDEV-7241</a>] -         Quantile and Tweedie distributions added to monotone constraints.
```
Comment (maurever) [inline Changes.md:25]: So can be Tweedie mention here or not if this feature was added in some previous fix release? @michalkurka

File: Changes.md [hunk 2]
```diff
@@ -2,6 +2,98 @@
 
 ## H2O
 
+### Zermelo (3.32.0.1) - 10/xx/2020
+
+Downlaod at: <a href='http://h2o-release.s3.amazonaws.com/h2o/rel-zermelo/1/index.html'>http://h2o-release.s3.amazonaws.com/h2o/rel-zermelo/1/index.html</a>
+
+#### Bug
+<ul>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-5932'>PUBDEV-5932</a>] -         Added support for Python 3.7+.
```
Comment (honzasterba) [inline Changes.md:11]: this is an improvement

File: Changes.md [hunk 3]
```diff
@@ -2,6 +2,98 @@
...
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7667'>PUBDEV-7667</a>] -         Fixed StackedEnsemble’s retrieval of the seed parameter value.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7746'>PUBDEV-7746</a>] -         Deserialization values of MOJO ModelParameter now work when the Value Type is int().
```
Comment (honzasterba) [inline Changes.md:15]: should be int[], or integer array

File: Changes.md [hunk 4]
```diff
@@ -2,6 +2,98 @@
...
+<li>
+[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7740'>PUBDEV-7740</a>] -         RuleFit model represented by the set of rules obtained from trees during training.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7765'>PUBDEV-7765</a>] -         Performance improved for exporting a Frame to CSV.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7769'>PUBDEV-7769</a>] -         GPU backend allowed in XGBoost when running multinode even when `build_tree_one_node` is enabled.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7787'>PUBDEV-7787</a>] -         Updated all URLs in R package to use HTTPS.
+</li>
+<li>[<a href='https://0xdata.atlassian.net/browse/PUBDEV-7790'>PUBDEV-7790</a>] -         Uprgraded to XGBoost 1.2.0.
```
Comment (honzasterba) [inline Changes.md:67]: typo

File: Changes.md (hunk 1 above)
Comment (michalkurka) [reply Changes.md:25]: yeah, it was already mentioned previously - we don't need to mention again

**Pull request merged**